CryptographyCodingExerciseWeek13/CryptographyCodingExerciseWeek13Test/UnitTest1.cs

```diff
@@ -0,0 +1,73 @@
+﻿using System;
+using System.IO;
+using System.Linq;
+using System.Security.Cryptography;
+using System.Security.Permissions;
+using System.Security.Principal;
+using System.Text;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using CryptographyCodingExerciseWeek13;
+
+namespace CryptographyCodingExerciseWeek13Test
+{
+    /// <summary>
+    /// Classt that tests the different types of encryption.
+    /// </summary>
+    [TestClass]
+    public class UnitTest1
```

Comment:
You need to create one test class for each class you are testing, using test class names like "SymmetricTests" and "DigitalSigning" tests. (-0.1)